superset-frontend/src/SqlLab/components/TableElement/index.tsx

```diff
@@ -72,41 +68,14 @@ const Fade = styled.div`
   opacity: ${(props: { hovered: boolean }) => (props.hovered ? 1 : 0)};
 `;
 
-const StyledCollapsePanel = styled(Collapse.Panel)`
-  ${({ theme }) => css`
-    & {
-      .ws-el-controls {
-        margin-right: ${-theme.sizeUnit}px;
-        display: flex;
-      }
-
-      .header-container {
-        display: flex;
-        flex: 1;
-        align-items: center;
-        width: 100%;
-
-        .table-name {
-          white-space: nowrap;
-          overflow: hidden;
-          text-overflow: ellipsis;
-          font-size: ${theme.fontSizeLG}px;
-          flex: 1;
-        }
-
-        .header-right-side {
-          margin-left: auto;
-          display: flex;
-          align-items: center;
-          margin-right: ${theme.sizeUnit * 8}px;
-        }
-      }
-    }
-  `}
+const StyledControls = styled.div`
```

Comment:
Don't want to go too much outside of the scope of the PR but do you think we can use standard Ant Design `Flex` and `Space` components to manage the layout for all the styles in this file?